app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardController.kt

```diff
@@ -438,6 +447,33 @@ class DashboardController(
                 .addTo(modelList)
         }
 
+        if ((ownHealthStatus is HealthStatusData.SelfTestingSuspicionOfSickness ||
+                    ownHealthStatus is HealthStatusData.SicknessCertificate) &&
```

Comment:
Use brackets to explain order.

Reply:
We have brackets, the OR in brackets will be computed and on the result will be an AND